tests/integration/test_loki_push_api.py

```diff
@@ -72,8 +72,8 @@ async def test_rule_files_ingested_by_loki(ops_test):
     await ops_test.model.wait_for_idle(status="active", timeout=1000)
 
     # now, make sure rules are present
+    expected = {"HighThroughputLogStreams", "HighPercentageError", "http-credentials-leaked"}
     response = await client.rules()
     assert (await client.rules()).items() > {}.items()
-    alert_group = next(iter(response.values()))[0]
-    assert "name" in alert_group
-    assert "rules" in alert_group
+    alerts = [group["rules"][0]["alert"] for group in next(iter(response.values()))]
```

Comment:
Now that we're checking for presence of alert names by name, the code must be more sophisticated than `rules[0]` - a rules group is a list of alerts. I.e. in a single file there can be multiple group names under `groups` and multiple alert names under `rules`.

Reply:
Done.